manifests/p/PatrickMortara/WIA-Loader/2.6.2.0/PatrickMortara.WIA-Loader.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: PatrickMortara.WIA-Loader
+PackageVersion: 2.6.2.0

```

```suggestion
PackageVersion: WIA-Loader 2.6.2.0
```
This must match Control Panel.